### ISSUE TITLE
Font smoothing doesn't match designs

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#11425]
Our new designs in Figma feel much lighter and airier than the reality in Chrome (on retina macOS). This looks to be because Figma runs with `-webkit-font-smoothing: antialiased` rather than `-webkit-font-smoothing: subpixel-antialiased` on retina screens, so @nadonomy has been picking weights/font-sizing and generally balancing the design in Figma... but the end result is then mangled by the browser. 

TL;DR: we should try setting everything to `-webkit-font-smoothing: antialiased` and its -moz equivalent. Need to check it doesn't make this illegible on Windows, and on non-retina displays. (I think `antialiased` is the default for non-retina displays though, so it hopefully shouldn't break those)

**Comment by ara4n** [comment]
Figma: (with experimental timeline font sizing; please ignore):
![figma](https://user-images.githubusercontent.com/1294269/69146115-0bbe1e00-0ac7-11ea-9d6c-1c76d48ef566.jpg)

Reality in Chrome on Retina on macOS:
![reality](https://user-images.githubusercontent.com/1294269/69146120-0eb90e80-0ac7-11ea-8f68-0bacca546507.jpg)

Using font-smoothing: antialiased on retina macOS rather than the default (subpixel-antialiased):
![reality-fixed](https://user-images.githubusercontent.com/1294269/69146126-0fea3b80-0ac7-11ea-9c17-f7b2961bae0d.png)

**Comment by ara4n** [comment]
(i'm currently running in Chrome with `-webkit-font-smoothing: antialiased` on the body tag, and it's a marked improvement on Retina in terms of feeling lighter and crisper and prettier).

It's worth noting that we've tried playing with this in the past and screwed it up as per https://github.com/matrix-org/matrix-react-sdk/commit/30bb7eeeae208fd05b57bbdda0e0e51a37831735 - it's unclear how/where the screwup is coming from this time.  Perhaps it's that Sketch used one smoothing and Figma used the other, and so having rebalanced the font weighting in Figma, that's how we've got out of whack.

**Comment by jryans** [comment]
MDN [claims](https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth) `-webkit-font-smoothing` / `-moz-osx-font-smoothing` only affect macOS, so Windows / Linux should be unaffected.

**Comment by turt2live** [comment]
Yea, before (which is just `auto` in our app) and after on Windows it's still a ransom note:
![image](https://user-images.githubusercontent.com/1190097/69167624-803a9200-0ab2-11ea-8e90-14957cd2d318.png)
![image](https://user-images.githubusercontent.com/1190097/69167629-83358280-0ab2-11ea-9caf-997d8f7e6dd0.png)

**Comment by ara4n** [comment]
@turt2live can you try disabling 'Accelerated 2D canvas' in chrome://flags?

**Comment by turt2live** [comment]
Presumably fixed by https://github.com/matrix-org/matrix-react-sdk/pull/3677
